tests/test_news.py

```diff
@@ -29,7 +29,7 @@ def check_news_file(fname):
                 pytest.fail(
                     "{}:{}: single grave accents"
                     " are not valid rst".format(name, i + 1),
-                    pytrace=False,
+                    pytrace=True,
```

Comment:
I actually didn't mean to commit this file, but it may not be a bad idea since the reason I altered the file was to save myself time debugging my mistakes with the `.rst` file I created. I'm sure I won't be the last